concord-ui/src/components/systemstatstable.js

```diff
@@ -0,0 +1,249 @@
+                        }}
+                        onClick={() => this.clearVotes(deadLockedPhrases[i].phrase.phraseId)}
+                      >
+                        Clear Votes
+                      </Button>
+                    </div>
+                    <div>
+                      <Button
+                        unelevated
+                        style={{
+                          backgroundColor: '#E0E0E0',
+                          color: 'black',
+                          marginBottom: '0.5rem'
+                        }}
+                        onClick={() => this.resolvePhrase(deadLockedPhrases[i].phrase.phraseId, 'TRASH')}
+                      >
+                        Trash
+                      </Button>
+                    </div>
+                  </DataTableCell>
+                  <DataTableCell />
+                </DataTableRow>
+              ))}
+            </DataTableBody>
+          </DataTableContent>
+        </DataTable>
+      </div>
+    );
+  }
+
+  // hack to put line breaks in DataTable (while trying not to break words)
+  splitPhrase(phrase) {
+    const textWidth = this.context.measureText(phrase).width;
+    const availableWidth = window.innerWidth / 3;
+    const widthRatio = textWidth / availableWidth;
+    const maxLineLength = Math.trunc(phrase.length / widthRatio) - 1;
+
+    let words = phrase.split(' ');
+    // break any long words up
+    let i = 0;
+    while (i < words.length) {
+      if (words[i].length >= maxLineLength) {
+        words.splice(i, 1, this.splitWord(words[i], maxLineLength));
+        words = this.flattenArray(words);
+      }
+      i++;
+    }
+
+    let lines = [];
+    i = 0;
+    while (i < words.length) {
+      let line = '';
+      while (i < words.length && line.length + words[i].length <= maxLineLength) {
+        line += ' ' + words[i];
+        i++;
+      }
+      lines.push(line);
+    }
+    return lines;
+  }
+
+  splitWord = (word, splitLength) => {
+    let chunks = [];
+    for (let i = 0, charsLength = word.length; i < charsLength; i += splitLength) {
+      chunks.push(word.substring(i, i + splitLength));
+    }
+    return chunks;
+  };
+
+  flattenArray = (a) => [].concat.apply([], a);
+
+  resolvePhrase(phraseId, label) {
+    // remove the phrase from the deadlocked list here (to save an expensive API call)
+    // for up to date information (e.g. if multiple users are resolving phrases) use the refresh button
+    this.removeDeadLockedPhrase(phraseId);
+    this.props.dispatch(resolveForPhraseLabel(phraseId, label));
+  }
+
+  removeDeadLockedPhrase(phraseId) {
+    let statsData = this.state.statsData;
+    statsData.deadLockedPhrases = statsData.deadLockedPhrases.filter((p) => p.phrase.phraseId !== phraseId);
+    this.setState({ statsData: statsData });
+  }
+
+  clearVotes(phraseId) {
+    this.removeDeadLockedPhrase(phraseId);
+    this.props.dispatch(deleteVotesForPhrase(phraseId));
+  }
+}
+
+export default connect((state) => ({
```

Comment:
export default connect(({ systemStats, nextPhrase }) => ({
  error: systemStats.error,
  loading: systemStats.loading,
  statsData: systemStats.statsData,
  loadingResolvePhrase: nextPhrase.loading,
  errorResolvePhrase: nextPhrase.error
}))(SystemStatsTable);

would be a bit nicer.